glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -1,4 +1,4 @@
-$schema: http://json-schema.org/draft-06/schema#
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
Is this new schema version supported on the pipeline side?

Reply:
It doesn't need to be -- this is only the schema for the `metrics.yaml` file, which is validated in this library and where we have control over the version of `jsonschema` used.

Reply:
Good, thanks for clarifying!